assets/app/data/crowbar/crowbar.constants.js

```diff
@@ -3,16 +3,16 @@
     angular
         .module('suseData.crowbar')
         .constant('UPGRADE_STEPS', {
-            upgrade_prechecks: 'upgrade_prechecks',
-            upgrade_prepare: 'upgrade_prepare',
-            admin_backup: 'admin_backup',
```

Comment:
you need to also update the errors section in the language file (rebase if needed to get latest changes).